gulpfile.js

```diff
@@ -949,28 +976,10 @@ gulp.task('fonttest', function () {
   return createTestSource('font');
 });
 
-gulp.task('botmakeref', function (done) {
-  console.log();
-  console.log('### Creating reference images');
+gulp.task('makeref', makeRef);
```

Comment:
nit: can you for symmetry wrap this call into the function. (and I don't know how gulp detects if the done `param` presents.)